dev/benchmarks/metrics_center/lib/src/github_helper.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file uses Dart 2.12 semantics. This is needed as we can't upgrade
+// the SDK constraint to `>=2.12.0-0` before the deps are ready.
+// @dart=2.12
```

Comment:
Why does this file need to use 2.12? If you are depending on un-migrated code there are some surprising semantics

Reply:
I guess that I'm blindly following https://github.com/flutter/flutter/commit/168ad5dc4d24c5d3bb3ab2e75ee192c8a4a8bbcf to avoid potential NNBD issues. Let me see if I can take this out.

Reply:
![Screen Shot 2020-11-12 at 6 13 47 PM](https://user-images.githubusercontent.com/22987568/99020201-d51cb700-2512-11eb-979e-054e8232dac1.png)

I got the error above as soon as I remove this. Do you know a better way to fix it other than `@dart=2.12`?

Reply:
You should not use nnbd language features until all of your dependencies are migrated, so in this case removing the nullable types should be sufficient.

Reply:
Removed.